build.cake

```diff
@@ -0,0 +1,268 @@
+#tool nuget:?package=NUnit.ConsoleRunner&version=3.4.1
+
+//////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+//////////////////////////////////////////////////////////////////////
+
+var target = Argument("target", "Default");
+var configuration = Argument("configuration", "Debug");
+
+//////////////////////////////////////////////////////////////////////
+// SET PACKAGE VERSION
+//////////////////////////////////////////////////////////////////////
+
+var version = "3.5.0";
+var modifier = "";
+
+var isAppveyor = BuildSystem.IsRunningOnAppVeyor;
+var dbgSuffix = configuration == "Debug" ? "-dbg" : "";
+var packageVersion = version + modifier + dbgSuffix;
+var packageName = "NUnit3TestAdapter-" + packageVersion;
+
+//////////////////////////////////////////////////////////////////////
+// DEFINE RUN CONSTANTS
+//////////////////////////////////////////////////////////////////////
+
+// Directories
+var PROJECT_DIR = Context.Environment.WorkingDirectory.FullPath + "/";
+var ADAPTER_DIR = PROJECT_DIR + "src/NUnitTestAdapter/";
+var TEST_DIR = PROJECT_DIR + "src/NUnitTestAdapterTests/";
+var INSTALL_DIR = PROJECT_DIR + "src/NUnitTestAdapterInstall/";
+var DEMO_DIR = PROJECT_DIR + "demo/";
+var PACKAGE_DIR = PROJECT_DIR + "package/";
+var PACKAGE_IMAGE_DIR = PACKAGE_DIR + packageName + "/";
+var TOOLS_DIR = PROJECT_DIR + "tools/";
+
+// TODO: Consolidate in one directory if possible
+var ADAPTER_BIN_DIR = ADAPTER_DIR + "bin/" + configuration + "/";
+var TEST_BIN_DIR = TEST_DIR + "bin/" + configuration + "/";
+var INSTALL_BIN_DIR = INSTALL_DIR + "bin/" + configuration + "/";
+var DEMO_BIN_DIR = DEMO_DIR + "NUnitTestDemo/bin/" + configuration + "/";
+
+// Solutions
+var ADAPTER_SOLUTION = PROJECT_DIR + "NUnit3TestAdapter.sln";
+var DEMO_SOLUTION = DEMO_DIR + "NUnit3TestDemo.sln";
+
+// Test Runners
+var NUNIT3_CONSOLE = TOOLS_DIR + "NUnit.ConsoleRunner/tools/nunit3-console.exe";
+
+// Test Assemblies
+var ADAPTER_TESTS = TEST_BIN_DIR + "NUnit.VisualStudio.TestAdapter.Tests.dll";
+var DEMO_TESTS = DEMO_BIN_DIR + "NUnit3TestDemo.dll";
+
+// Packages
+var ZIP_PACKAGE = PACKAGE_DIR + "NUnit3TestAdapter-" + packageVersion + ".zip";
```

Comment:
Good point. This will all be revised once we settle some of the issues of using GitVersion, but it should be consistent for now. The other package names are set after `InitializeBuild` takes effect, so packageVersion has changed. This one, being pre-initialized gets the original default setting.